### PR TITLE
[Doc Improvement][Call recording and meeting][3993045]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4faef140bc0b23c41558aa15408d1e350d17580e/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 93 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f34e04f6-dffb-4e1e-8e62-b9e4e1c40855) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c8747ecd16969911848a1e39f0c0ec26b98bca3/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c0300442-3532-4249-8e62-0f99fa59978c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/837f1a5434a835c75753447d6a01b32952d64e28/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 93 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38456f07-68f9-43c0-9532-37084e85c18f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c995bdc70a988414aeace1f129a32da30c21cb1b/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 93 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e98286b5-7cf4-49ae-b4dc-be2bbb6d354b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3aa366283bf2b79b0a1cffa0d551de9e9ea72b93/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 93 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d60a1b08-81a9-49ca-a0b5-9a2edd3421fd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b82eaa21c491ecfce3ba55898195ba9c8be662a0/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d1bb435-e774-4d84-8452-f5721d81374a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by devjha-ms — APPROVED** [review]
Changes looks good.

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 1]
```diff
@@ -48,6 +49,15 @@ For more information about the meeting-specific RSC permissions, see [RSC permis
 
 After you've configured the permissions, configure your app to receive [change notifications](/graph/teams-changenotifications-callrecording-and-calltranscript) for transcripts and recordings available events or all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content and recording. Your app can fetch the transcript and recording for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` file. The recording of the meeting is available as an `.mp4` file.
 
+You can use the following types of change notification for your app:
+
+| Use | When... |Supported resource paths|
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:54]: Can we consider changing the table headings to | Subscription Scope | Description | Supported resource paths |

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 2]
```diff
@@ -48,6 +49,15 @@ For more information about the meeting-specific RSC permissions, see [RSC permis
 
 After you've configured the permissions, configure your app to receive [change notifications](/graph/teams-changenotifications-callrecording-and-calltranscript) for transcripts and recordings available events or all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content and recording. Your app can fetch the transcript and recording for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` file. The recording of the meeting is available as an `.mp4` file.
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:50]: Can we add a ## heading 'Get notified when a transcript or recording is available' before this paragraph?

**Review by Wajeed-msft — APPROVED** [review]
Please fix all the open comments before merging.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dae23903a91351a894595d3bfff27e2173d9cba5/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0afda73-ce19-4b6a-b1ed-df30d203b219) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 2 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:50]: Updated as suggested.

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 1 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:54]: Updated as suggested

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/665afc55c13194476c0c319fcec1698444aa7767/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a9529ae-5ee1-4870-8308-84757cfff186) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Anjali-Patle — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/baf03fc26d047456bf112d17bd5e7a76f103d309/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba96b210-57af-4e8d-81ca-d79ffcda527c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**